src/hooks/usePatchStatusSelect.ts

```diff
@@ -63,41 +71,47 @@ type UpdatedPatchBuildVariantType = Omit<GroupedBuildVariant, "tasks"> & {
     baseStatus?: string;
   }[];
 };
+
+type ChildVersions = {
+  id: string;
+  buildVariants?: UpdatedPatchBuildVariantType[];
+}[];
 export const usePatchStatusSelect = (
```

Comment:
Maybe we should rename this to `useVersionTaskStatusSelect`. What do you think?

Reply:
added to https://jira.mongodb.org/browse/EVG-15414